internal/server/scriptlet/qemu.go

```diff
@@ -41,6 +42,22 @@ func marshalConf(conf any) ([]map[string]any, error) {
 	return newConf, nil
 }
 
+// unmarshalConf unmarshals a configuration into a []cfgSection.
+func unmarshalConf(conf any) ([]cfgSection, error) {
```

Comment:
Same with this function.

Actually would it make sense to have those in the `cfg` package instead?

Reply:
I don’t think so. I see no other place where using this function would make sense, so let’s keep it here to really mean that it’s only supposed to be used here!